Formula/ansible-lint.rb

```diff
@@ -124,6 +86,10 @@ class AnsibleLint < Formula
 
   def install
     virtualenv_install_with_resources
+    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+    site_packages = "libexec/lib/python#{xy}/site-packages"
+    ansible = Formula["ansible"].opt_prefix
+    (prefix/site_packages/"homebrew-ansible.pth").write ansible/site_packages
```

Comment:
Doesn't this need to be something like
```ruby
(prefix/site_packages/"homebrew-ansible.pth").write "import site; site.addsitedir('#{ansible/site_packages}')"
```
?
You can probably also simplify this a bit by defining
```ruby
ansible = Formula["ansible"].opt_libexec
```
instead.

Reply:
It doesn't seem necessary to execute `site.addsitedir()` in a `.pth` file [doc](https://docs.python.org/3/library/site.html).

Just adding the path worked for me, I could see the path when executing the venv python and ansible-lint did execute.

Reply:
Huh, weird. TIL.